src/qt/lookupspdialog.h

```diff
@@ -39,6 +41,9 @@ public Q_SLOTS:
     Ui::LookupSPDialog *ui;
     WalletModel *model;
 
+    GUIUtil::TableViewLastColumnResizingFixer *borrowedColumnResizingFixer;
```

Comment:
Variable name hard to understand. IMO rename to `denominationTableLastColFixer` will be more better.